src/settings.rs

```diff
@@ -48,6 +48,8 @@ pub struct Client {
     /// Whether aliases specified in `pueue_aliases.yml` should be expanded in the `pueue status`
     /// or shown in their short form.
     pub show_expanded_aliases: bool,
+    /// Whether the client should use dark shades instead of regular colors.
+    pub dark_mode: bool,
```

Comment:
My plan with 0.12 and all upcoming releases is to stay backward compatible.
I.e. previous states don't break on a version update.

For this, we need serde default options, which are sadly kind of ugly in their current state.

The code for that looks like this:
```
    /// Whether the client should use dark shades instead of regular colors.
    #[serde(default = "default_dark_mode")]
    pub dark_mode: bool,
```

And this function, which should be located somewhere in `settings.rs`.
```
/// The default value for the `dark_mode` client settings.
/// Needed to keep backward compatibility between v0.11 and v0.12
fn default_dark_mode() -> bool {
    false
}
```

If more of those default functions pop up, they'll be moved into a separate module.